marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mapping/getNewDocument.api

```diff
@@ -0,0 +1,18 @@
+{
+    "functionName": "getNewDocument",
```

Comment:
This endpoint is returning a JSON object with 3 things - the actual document, an optional namespaces object, and a format string. I think we need a noun other than "document" here, as "document" implies you're just getting back a document, but you're getting more than that. 

The use case is - I want to test my mapping against a document. How about "getDocumentForTesting"? I think that gives us the semantic flexibility to return a JSON object where "document" is one property (as opposed to "data", which is too vague).

Reply:
Changed to getDocumentForTesting.  Endpoint for front end also changed to `testingDoc`.